docs/download-upload-clone/development-plan.md

```diff
@@ -0,0 +1,71 @@
+Each phase assumes that given version of exported feature is fully importable.
+
+# Release 1
+
+* zip file generation
+* zip file storage
+* internal zip file format with versioning set
+* blocking project/scenarios actions during export
+* export project's:
+    * metadata (name etc)
+* export underlying scenarios with:
+    * metadata (name etc)
+
+# Release 2
+
+* project's:
+	* planning area: selected GADM & grid config settings
+	* custom planning area: selected GADM & grid config settings (first 
+	  shapefile handling)
+	* generated PU with geometries
+
+# Release 3
+
+* project's:
+	* custom grid, settings and generated PU with geometries
```

Comment:
in case of custom PU grids, all we may have is the source shapefile (if we decide to store it), its name (we would need to store it if we want to use it), and the PU geometries persisted to db from the shapefile we originally processed; there would be no other settings involved here, AFAICT.
the PU geometries in this case would not be generated, strictly speaking, but persisted from the shapefile input (after any cleanup such as `st_makevalid()` as for other custom uploads such as custom features)